src/ResourceManager/Network/Commands.Network/help/Get-AzureRmVirtualHub.md

```diff
@@ -0,0 +1,133 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/get-azurermvirtualhub
+schema: 2.0.0
+---
+
+# Get-AzureRmVirtualHub
+
+## SYNOPSIS
+Gets an Azure VirtualHub by Name and ResourceGroupName or lists all Virtual Hubs by ResourceGroupName/Subscription.
+
+## SYNTAX
+
+### ListByResourceGroupName (Default)
+```
+Get-AzureRmVirtualHub -ResourceGroupName <String> [-Name <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### ListBySubscriptionId
+```
+Get-AzureRmVirtualHub [-ResourceGroupName <String>] [-Name <String>] [-DefaultProfile <IAzureContextContainer>]
```

Comment:
This should have no parameters, rather than both resource group name and name optional.  Apply to all gets.

Reply:
This is auto generated. I used two parameter sets one for sub id and one for RG name. I am not sure how to have no parameters.

Reply:
Answered this question in the cmdlets themselves.  Please be sure to regenerate all files to get this change in.